iosHackthonApp/Modules/Features/Login/Sources/Login/LoginView.swift

```diff
@@ -46,16 +53,21 @@ public struct LoginView: View {
 						Text(Strings.Login.email)
 						TextField(Strings.Login.emailPlaceholder, text: $email)
 							.modifier(GreyTextFieldStyle())
+							.autocapitalization(.none)
 						Text(Strings.Login.password)
 						SecureField(Strings.Login.passwordPlaceholder, text: $password)
 							.modifier(GreyTextFieldStyle())
 						Spacer()
 
-
 						GeometryReader { geometry in
+							NavigationLink(destination: TabAppView(selectedView: 0), isActive: $loginSucessful) {
```

Comment:
I would like to use this here `isActive: authorizationStore.isUserAuthorized()` instead of  `isActive: $loginSucessful` and get rid of the extra variable `loginSucessful`.
 But I m not sure how to do this as `isActive` is expecting a binding value.

Reply:
Hmm. 
Might take a bit of work, it would definitely be easier if we got rid of the protocol `authorizationStoreType` and then we could make `AuthorizationStore` conform to `ObservableObject` and just have a `@Published var isAuthorised = false` and use that as the binding - you can update it from your `isAuthorized()` function, set the authorizationStore property in this file to be `@observedObject private var authorizationStore` and I guess that would work 🤔

Reply:
I haven't tried any of that, but let me know if it doesn't make sense or if you need a hand trying it :-)

Reply:
Sounds like that would work.. let me try 🙂

Reply:
Got it working!😄